content/docs/conditional-rendering.md

```diff
@@ -116,13 +120,16 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
```

Comment:
Interesting inconsistency in the English docs: in other pages, they don't use the `**` and so far we haven't used them either. So maybe to stay consistent in the Dutch translation, don't use `**` for these links?

Reply:
@ronderksen Yes, the early articles in the docs use empty links like `[](codepen.io/link)`. However, I personally think the fat links `[**Try it on CodePen**](codepen.io/link)`go better with the overall layout of the page. I would be in favor of adapting earlier articles to the latter format.